src/commands/bridge/nft.ts

```diff
@@ -176,16 +176,20 @@ export default class BridgeNFT extends Command {
     CliUx.ux.action.start('Retrieving collection smart contract')
     const collectionABI = abis.HolographDropERC721ABI
     const collection = new Contract(collectionAddress, collectionABI, this.networkMonitor.providers[sourceNetwork])
+    console.log(`Collection address is ${collection.address}`)
+
     CliUx.ux.action.stop()
 
     this.log(`tokenId is ${tokenId}`)
     CliUx.ux.action.start('Checking if token ID exists on ' + sourceNetwork + ' network.')
-    const tokenExists: boolean = await collection.exists(tokenId)
-    CliUx.ux.action.stop()
-    if (!tokenExists) {
-      this.log('Token does not exist.')
-      this.exit()
-    }
+
+    // TODO: This only works for legacy collections and not drops
```

Comment:
I suppose we need a check to so we can bridge either drops or legacy nfts.

Reply:
not sure how to do this at the moment as i don't think we have a similar function on the drop contracts. open to ideas though @ACC01ADE maybe you have some thoughts?

Reply:
Can discuss it on a call and figure it out.

Reply:
But we should have that function available to us in drops as well since it inherits the same enforcer contract type.

Reply:
reenabled this